### DISCUSSION TITLE
Alpha

### DISCUSSION THREAD

**Discussion by vurt72** [BadDogSkyrim/PyNifly#160] [General]
How can i display alpha properly, for example when you have leaves on a tree and you only want to see the leaves. 
thanks

**Comment by BadDogSkyrim** [comment]
Do you mean in Blender? It depends on a bunch of things. If you use "materials" viewport shading with "Combined" render pass then objects with transparency that you bring in with pynifly should show transparency.

**Comment by vurt72** [comment]
Thank you! I figured it out.
Very thankful for this addon btw, it's really great :)
